lale/lib/lale/hyperopt.py

```diff
@@ -263,27 +263,14 @@ def get_trials(self):
 _input_fit_schema = {
     'type': 'object',
     'properties': {
-        'X': {
-            'type': 'array',
-            'items': {
-                'anyOf': [
-                {   'type': 'array', 'items': {'type': ['number', 'string']}},
-                {   'type': 'string'}]}},
-        'y': {
-            'type': 'array', 'items': {'type': 'number'}}}}
-
+        'X': {},
+        'y': {}}}
 _input_predict_schema = {
     'type': 'object',
     'properties': {
-        'X': {
-            'type': 'array',
-            'items': {
-                'anyOf': [
-                {   'type': 'array', 'items': {'type': ['number', 'string']}},
-                {   'type': 'string'}]}}}}
+        'X': {}}}
 
-_output_predict_schema = {
-    'type': 'array', 'items': {'type': 'number'}}
+_output_predict_schema:Dict[str, Any] = {}
```

Comment:
That makes sense, since we don't know a-priori what the output schema will be. In the future, we may need to add a `transform_schemas` method so that the output schema of hyperopt is the same as the output schema of the pipeline that it wraps.